### PR TITLE
Link to Malli presentation added to docs

### PR DESCRIPTION
**Pull request by vkjr**
### Summary

Link to Malli presentation from @ilmotta added to docs

### CODE REVIEW

**Review by smohamedjavid — APPROVED** [review]

**Review by ajayesivan — APPROVED** [review]

**Review by clauxx — APPROVED** [review]

**Review by Parveshdhull — APPROVED** [review]

File: doc/README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@
 
 [Contributing to status-go](status-go-changes.md)
 
+[Malli schemas (recorded demo)](https://drive.google.com/file/d/1orqa8Ky2mVqGJCLksgRF7F5hqjurMxt0/view)
```
Comment (siddarthkay) [inline doc/README.md]: This link is private and probably of no use to folks that do not have a status email address.
Should we consider uploading this presentation to Status Youtube and then link it here ?
Also a separate link to PPTs will also be nice

Comment (vkjr) [reply doc/README.md]: Good idea. But I'm not sure who has permission to upload videos to youtube. @cammellos, maybe you know?

@ilmotta, would you mind to share the slides?

Comment (siddarthkay) [reply doc/README.md]: I've asked in Logos Discord

Comment (siddarthkay) [reply doc/README.md]: We can post on youtube, I'm waiting to hear from @ilmotta on his thoughts 😅

Comment (ilmotta) [reply doc/README.md]: @siddarthkay @vkjr it's fine by me. I think the presentation doesn't contain anything egregiously wrong and if it helps anyone why not share, right? I'll add the slides to the same Status Demo directory in G. Drive. Thanks guys :) 

Edit: The PDF is in `Status Drive` > `Development` > `Demos`

**Review by ilmotta — APPROVED** [review]

File: doc/README.md (hunk 1 above)
Comment (siddarthkay) [reply doc/README.md]: Thanks @ilmotta : This will be uploaded Monday next week.

Comment (siddarthkay) [reply doc/README.md]: here @vkjr -> https://www.youtube.com/watch?v=SlRio70aYVI

**Pull request merged**